jobs/haproxy/templates/pre-start.erb

```diff
@@ -16,3 +16,7 @@ fi
 if [ ! -e /usr/local/bin/hatop ]; then
   sudo ln -s /var/vcap/packages/haproxy/hatop-wrapper /usr/local/bin/hatop
 fi
+
+if [ ! -e /usr/local/bin/socat ]; then
+  sudo ln -s /var/vcap/packages/haproxy/socat /usr/local/bin/socat
```

Comment:
Hi @maxmoehl , only `/var/vcap/packages/haproxy/bin/socat` exists, this works for me:
```
sudo ln -s /var/vcap/packages/haproxy/bin/socat /usr/local/bin/socat
```

The symlink creation of this PR does not fail but points into nowhere. The acceptance tests could be more expressive but I think that they are catching such an issue at all is already good enough :)

Reply:
🤦🏻‍♂️ Thanks!

Reply:
Classic Copy-Paste issue 😄